### PR TITLE
Return new `fileName` field on `martha_v3` WA-344

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Review by rsasch — APPROVED** [review]

**Review by salonishah11 — APPROVED** [review]

**Comment by kshakir** [comment]
- broke the integration tests with a dependency update
- fixed the integration tests
- added a documented `ENV=mock npm run integration` to simplify integration testing outside of FiaBs

File: test/martha/smoketest_v2.test.js [hunk 1]
```diff
@@ -6,7 +6,8 @@
  */
```
Comment (salonishah11) [inline test/martha/smoketest_v2.test.js:6]: We would still need to pass the `BASE_URL` here while running the smoke tests right? If not, we should update the comment here and in `smoketest_v3.test.js`

File: README.md [hunk 2]
```diff
@@ -260,6 +263,45 @@ Run the `npx` command using `--fix` flag: `npx eslint <file_name/directory_name>
 
 `npm test`
 
+### Run Integration Tests
```
Comment (salonishah11) [inline README.md:266]: 💯 🌟

File: common/config.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+// Start with the defaults...
+const configDefaults = {
+    samBaseUrl:
+        `https://sam.dsde-${terraEnv}.broadinstitute.org`,
+    bondBaseUrl:
+        (() => {
+            // noinspection JSUnreachableSwitchBranches
+            switch (marthaEnv) {
+            case ENV_MOCK: return 'http://127.0.0.1:8080';
+            default: return `https://broad-bond-${terraEnv}.appspot.com`;
```
Comment (rsasch) [inline common/config.js]: nit: indent?

File: common/config.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            }
+        })(),
+    dataObjectResolutionHost:
+        (() => {
+            // noinspection JSUnreachableSwitchBranches
+            switch (marthaEnv) {
+            case ENV_MOCK: return 'wb-mock-drs-dev.storage.googleapis.com';
+            case ENV_PROD: return 'gen3.biodatacatalyst.nhlbi.nih.gov';
+            default: return 'staging.gen3.biodatacatalyst.nhlbi.nih.gov';
```
Comment (rsasch) [inline common/config.js]: nit: indent?

File: common/config.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+            }
+        })(),
+    itMarthaBaseUrl:
+        (() => {
+            // noinspection JSUnreachableSwitchBranches
+            switch (marthaEnv) {
+            case ENV_MOCK: return 'http://localhost:8010';
+            default: return `https://martha-fiab.dsde-${terraEnv}.broadinstitute.org:32443`;
```
Comment (rsasch) [inline common/config.js]: nit: indent

File: common/config.js [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+            }
+        })(),
+    itBondBaseUrl:
+        (() => {
+            // noinspection JSUnreachableSwitchBranches
+            switch (marthaEnv) {
+            case ENV_MOCK: return 'http://127.0.0.1:8080';
+            default: return `https://bond-fiab.dsde-${terraEnv}.broadinstitute.org:31443`;
```
Comment (rsasch) [inline common/config.js]: nit: indent?

**Review by rsasch — APPROVED** [review]
small nits, otherwise LGTM 👍

File: common/config.js (hunk 3 above)
Comment (kshakir) [reply common/config.js]: Existing lint configuration said no. I'll take a really quick look and see if I can find the eslint rule to fix this though.

File: test/martha/smoketest_v2.test.js (hunk 1 above)
Comment (kshakir) [reply test/martha/smoketest_v2.test.js:6]: Yes, the `BASE_URL` is needed for smoke testing.

**Review by salonishah11 — APPROVED** [review]

File: common/config.js (hunk 3 above)
Comment (kshakir) [reply common/config.js]: Found it!

**Pull request merged**